frontend/assets/javascripts/src/modules/metrics/forms.js

```diff
@@ -0,0 +1,80 @@
+define([
+    'bean',
+    'src/modules/metrics/logger',
+    'src/modules/form/helper/formUtil'
+], function (bean, logMetric, formUtil) {
+
+    function init() {
+        if (window.ga && formUtil) {
+            trackForms();
+        }
+    }
+
```

Comment:
potentially add some comments in here? whats being logged - what the output will be? May help with future clarity?